.github/workflows/bypass-review.yaml

```diff
@@ -25,8 +25,15 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.ADMIN_PAT }}
         run: |
+          # Get the author of the PR
           AUTHOR=$(gh pr view ${{ github.event.pull_request.number }} --json author --jq '.author.login')
-          ADMIN_CHECK=$(gh api orgs/scidsg/members/$AUTHOR --jq '.role')
+
+          # Check the user's role using memberships API (works for both public and private members)
+          ADMIN_CHECK=$(gh api orgs/scidsg/memberships/$AUTHOR --jq '.role')
+
+          # Output the result for debugging purposes
+          echo "Admin check response: $ADMIN_CHECK"
```

Comment:
Do we need this? What if we included the user's role in the error message if it's not admin?

Reply:
👍 Good call. The role is included in the message now. 2d1e152